simplq/src/components/pages/Join/JoinPage.jsx

```diff
@@ -17,19 +18,19 @@ export default ({ history, match }) => {
 
   useEffect(() => {
     dispatch(getQueueStatusByName({ queueName }));
-  }, [queueName, dispatch, getQueueStatusByName, history]);
+  }, [queueName, dispatch, getQueueStatusByName]);
 
   const queueId = queueStatus.queueId;
 
   const joinQueueHandler = (name, contactNumber) => {
-    // TODO: Refactor JoinQueueForm to get state from redux.
-    // TODO: remove return
-    return dispatch(joinQueue({ name, contactNumber, notifiable: true, queueId }));
+    dispatch(joinQueue({ name, contactNumber, notifiable: true, queueId }));
   };
-
+  // TODO: If HeaderSection is used just in JoinPage
+  // it should be renamed into something else and moved
+  // closer to JoinPage
```

Comment:
We should think about this later. Is this to be solved using react components or by writing reusable CSS?

Reply:
Reusable component would probably be a good solution.

```jsx
<Header {title,  subtitle}></Header>
```